common/app/views/main.scala.html

```diff
@@ -27,7 +27,7 @@
 }
 
 <!DOCTYPE html>
-<html id="js-context" class="js-off is-not-modern id--signed-out" data-page-path="@request.path">
+<html id="js-context" class="js-off is-not-modern id--signed-out" lang="en" data-page-path="@request.path">
```

Comment:
Yeah that's one of the things I wanted to talk about on this pr. I don't even know where I'd locate or how the other possible languages are supported.

I googled "Guardian In Arabic" and I can see this https://www.theguardian.com/global/video/2016/jul/11/sudanese-artist-us-road-trip-civil-rights-video-arabic-version which is an article in Arabic, but the outer page itself is still English.